src/renderer/components/Map.tsx

```diff
@@ -62,17 +64,22 @@ function makeStationMarker(
   const diameter = circleDiameter(species.size(), maxSpecies);
   const maxPreviewPhotosCount = 3;
 
-  const markerElement = document.createElement('div');
-  markerElement.className = styles.marker;
```

Comment:
Let's remove the now-dead Sass code from `Map.module.scss`.

Reply:
Done :)